app/screens/profile/ProfileMainScreen.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {
+  Text, StyleSheet, FlatList, SafeAreaView,
+} from 'react-native';
+import SettingsButton from './SettingsButton';
+import { COLORS } from '../../constants/colors';
+import UserIcon from '../shared/UserIcon';
+import ReviewItem from './ReviewItem';
+
+const mockUser = {
```

Comment:
I know this is just temporary but try to separate mock data into a separate file and just import in here to shorten the amount of code in this component file. Same thing for mockReviews